doc/Design/Parts/Component Beagle Core.lyx

```diff
@@ -116,119 +116,264 @@ List all associated packages
 \end_layout
 
 \begin_layout Section
-The most important classes
+Overview
 \end_layout
 
-\begin_layout Paragraph
-The Controller classes.
+\begin_layout Subsection*
+Controller classes
 \end_layout
 
 \begin_layout Standard
 The classes 
-\begin_inset ERT
+\begin_inset listings
+inline true
 status open
 
 \begin_layout Plain Layout
 
-
-\backslash
-lstinline{Beagle Controller}
+Beagle Controller
 \end_layout
 
 \end_inset
 
  and 
-\begin_inset ERT
+\begin_inset listings
+inline true
 status open
 
 \begin_layout Plain Layout
 
+Measurement Controller
+\end_layout
+
+\end_inset
+
+ manage the invocation of 
+\begin_inset listings
+inline true
+status open
 
-\backslash
-lstinline{Measurment Controller}
+\begin_layout Plain Layout
+
+Measurement Tool
 \end_layout
 
 \end_inset
 
- manage, when which measurement or result analyser component is working.
+ or 
+\begin_inset listings
+inline true
+status open
+
+\begin_layout Plain Layout
+
+Result Analyser
 \end_layout
 
-\begin_layout Standard
-There is always only one measurement tool, one result analyser or the final
- judge working.
+\end_inset
+
+ components.
+ 
+\begin_inset listings
+inline true
+status open
+
+\begin_layout Plain Layout
+
+Beagle Controller#main
 \end_layout
 
-\begin_layout Standard
-The 
-\begin_inset ERT
+\end_inset
+
+ is the main control loop, managing the control flow throughout Beagle’s
+ measuring and analysis activity.
+ There is always exactly one 
+\begin_inset listings
+inline true
 status open
 
 \begin_layout Plain Layout
 
+Measurement Tool
+\end_layout
 
-\backslash
-lstinline{Beagle Controller}
+\end_inset
+
+, 
+\begin_inset listings
+inline true
+status open
+
+\begin_layout Plain Layout
+
+Result Analyser
+\end_layout
+
+\end_inset
+
+ or 
```

Comment:
I think `or [the]`... is better, as this makes clear, there is only one final judge.